Userland/Libraries/LibCompress/Deflate.h

```diff
@@ -103,7 +103,7 @@ class DeflateDecompressor final : public InputStream {
     CircularDuplexStream<32 * KiB> m_output_stream;
 };
 
-enum DeflateSpecialCodeLengths : u32 {
+enum class DeflateSpecialCodeLengths : u32 {
```

Comment:
Not sure I love this, the majority of users of the enum deal with the raw versions, meaning it's all static casts and to_underlying()s